.github/workflows/release.yml

```diff
@@ -0,0 +1,23 @@
+name: Release package
+
+on:
+  workflow_run:
+    workflows: [Run Tests]
+    types: [completed]
+    branches: [master]
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  release-if-needed:
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create Release PR
+        uses: google-github-actions/release-please-action@v4
+        id: release
+        with:
+          token: ${{ secrets.RELEASE_PLEASE_GITHUB_TOKEN }}
```

Comment:
This currently has my personal github token. @stympy can you please set it to a different value?
Perhaps the Personal Access Token of the _honeybadger-robot_ account? On this note, it may be a good idea to make an organizational secret so that we can use it in multiple repositories without having to create a secret for each repository.

Reply:
I have updated the token in the repo settings. If it works, then I can set it for the org.